main.js

```diff
@@ -36,6 +36,48 @@ var formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+function ODate(rootEl) {
```

Comment:
There is an unwritten rule that constructor functions begin with a capital letter :-)